scripts/lang_adapt/madx_run_clm.py

```diff
@@ -105,7 +115,7 @@ class ModelArguments:
     )
     lang_adapt_strategies: str = field(
         default="",
-        metadata={"help": "choose one of the three strategies - 'emb', 'emb-and-adpt', 'emb-then-adpt'"},
+        metadata={"help": "choose one of the three strategies - 'emb', 'emb-and-adpt', 'emb-then-adpt', 'emb-and-sft'"},
```

Comment:
right now I will remove `emb-and-*` as it should be the default. So `emb-and-sft` will become `sft`.